src/test/java/com/epam/tc/hw2/ex2/Exercise2.java

```diff
@@ -0,0 +1,85 @@
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import org.testng.annotations.Test;
+
+public class Exercise2 extends BeforeAfter {
+    WebDriverWait webDriverWait;
+    private WebElement webElement;
+    static final SoftAssertions SOF = new SoftAssertions();
+
+    @Test
+    public void exercise2() {
+        webDriverWait = new WebDriverWait(webDriver, 10);
+        webDriver.manage().window().maximize();
+        webDriver.navigate().to("https://jdi-testing.github.io/jdi-light/index.html");
+        webDriverWait.until(ExpectedConditions.urlToBe("https://jdi-testing.github.io/jdi-light/index.html"));
+        //Assert Browser title
+        webElement = webDriver.findElement(By.name("main-title"));
+        webDriverWait.until(ExpectedConditions.titleIs("Home Page"));
+        SOF.assertThat(webDriver.getTitle()).isEqualTo("Home page");
+        //Assert performing login
+        WebElement openLogin = webDriver.findElement(By.id("user-icon"));
+        openLogin.click();
+        webDriverWait.until(ExpectedConditions.elementToBeClickable(openLogin));
+        WebElement username = webDriver.findElement(By.id("name"));
+        WebElement password = webDriver.findElement(By.id("password"));
+        final WebElement LOG = webDriver.findElement(By.id("login-button"));
+        username.sendKeys("Roman");
+        password.sendKeys("Jdi1234");
+        LOG.click();
+        webDriverWait.until(ExpectedConditions.urlToBe("https://jdi-testing.github.io/jdi-light/index.html"));
+        SOF.assertThat(webDriver.getCurrentUrl()).isEqualTo("https://jdi-testing.github.io/jdi-light/index.html");
+        //        Assert User name in the left-top side of screen that user is loggined
+        WebElement usernameLoged = webDriver.findElement(By.id("user-name"));
+        usernameLoged.isDisplayed();
+        webDriverWait.until(ExpectedConditions.textToBePresentInElement(usernameLoged, "ROMAN IOVLEV"));
+        SOF.assertThat(usernameLoged.getText()).isEqualTo("ROMAN IOVLEV");
+        //Open through the header menu Service -> Different Elements Page
+        WebElement serviceOnTheHeader = webDriver.findElement(By.cssSelector(".dropdown-toggle"));
+        serviceOnTheHeader.click();
+        webDriverWait.until(ExpectedConditions.elementToBeClickable(serviceOnTheHeader));
+        WebElement differentElemPage = webDriver.findElement(By.xpath("//a[@href='different-elements.html']"));
+        differentElemPage.click();
+        webDriverWait.until(ExpectedConditions.urlToBe(
+            "https://jdi-testing.github.io/jdi-light/different-elements.html"));
+        SOF.assertThat(webDriver.getCurrentUrl()).isEqualTo(
+            "https://jdi-testing.github.io/jdi-light/different-elements.html");
+        //Select checkboxes
+        WebElement waterCheck = webDriver.findElement(By.xpath(
+            "//label[@class='label-checkbox'][1]//input[@type='checkbox']"));
```

Comment:
It's better to not use indexes in the xpath

Reply:
I fixed all the other comments, but with this I have problems. I spent a lot of time thinking how to find these four elements becasuse they have the same class and type. The difference only in text(Water,Wind...) but I don't know how to find them by this attribute. Can you help me, please)

Reply:
I can't see any other options in this case beside the xpath, but to remove indexes you could try to find those elements by text for example :)

Reply:
I understand that i can do it by text but i am trying to find it like "//*[contains(text(),‘Wat’)]" or "//label[@class='label-checkbox']//input[@type='checkbox']//*[contains(text(),‘Water’)]" and don't get results. May be i don't understand something and doing it wrong.

Reply:
Here's the example that worked for me: https://stackoverflow.com/questions/3655549/xpath-containstext-some-string-doesnt-work-when-used-with-node-with-more

Reply:
Thank you very much!)